Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.7
+
+#declare envs
```

Comment:
Importance of these declarations?

Reply:
The env fields are captured during the build process. Knowing these required fields prior makes it easier and safer for another dev to add the required values.